.editorconfig

```diff
@@ -0,0 +1,20 @@
+# This file is for unifying the coding style for different editors and IDEs
+# editorconfig.org
+
+# WordPress Coding Standards
+# https://make.wordpress.org/core/handbook/coding-standards/
+
+# Customized for 2 space indentation and Markdown files
+
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+indent_style = space
+indent_size = 4
+
+[{*.md,*.markdown}]
+trim_trailing_whitespace = false
```

Comment:
please remove this file, I've just applied https://github.com/sitrunlab/learn-kahlan/commit/136f034b048f1ee18d777f8668a2501fc0611b71 for styleCI automatic check. After that, please do rebase https://samsonasik.wordpress.com/2015/09/16/practical-git-4-rebasing-conflicted-task-branch-against-primary-branch/